### PR TITLE
Add rubygem-rsec package

### PR DESCRIPTION
**Pull request by witlessbird**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.15
* [ ] 1.14
* [ ] 1.13

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by lzap** [comment]
Just a heads up - I merged DHCP PR and we are good to go.

**Comment by witlessbird** [comment]
* fixed comps-foreman-rhel7.xml
* Updated the license in spec file
* added %license macro

File: rubygem-rsec/rubygem-rsec.spec [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+%gem_install
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/bench
```
Comment (domcleal) [inline rubygem-rsec/rubygem-rsec.spec]: Could you move this to the docs? Benchmarks aren't needed for runtime.

**Comment by witlessbird** [comment]
* moved benchmars under docs

**Comment by domcleal** [comment]
Merged and building, thanks @witlessbird.

**Pull request merged**